packer/preview-ami.pkr.hcl

```diff
@@ -57,9 +57,7 @@ build {
             "OKTA_API_KEY=${var.preview_okta_api_key}",
             "ENVIRONMENT=${var.environment}",
             "PREVIEW_TEALIUM_TAG=${var.preview_tealium_tag}",
-            "PREVIEW_TEALIUM_ENV=${var.preview_tealium_env}",
-            "PREVIEW_LD_API_KEY=${var.preview_ld_api_key}",
-            "PREVIEW_LD_CLIENT_ID=${var.preview_ld_client_id}"
+            "PREVIEW_TEALIUM_ENV=${var.preview_tealium_env}"
```

Comment:
why remove the launch darkly keys?

Reply:
They aren't used by the AMI creation process. I could add them, but they would still not get used. I chose to remove them to more accurately what is needed and is used by the AMI creation process.

Reply:
that makes sense, I like that